src/porepy/fracs/fracture_network_2d.py

```diff
@@ -411,39 +439,65 @@ def prepare_for_gmsh(
             self._insert_auxiliary_points(**mesh_args)
         else:
             self._set_mesh_size_without_auxiliary_points(**mesh_args)
-        # Transfer data to the format expected by the gmsh interface
+
+        # Transfer data to the format expected by the gmsh interface.
+        # This requires some information processing and translation between data
+        # formats: In the geometry processing undertaken up to this point, it has been
+        # convenient to use numerical values for identifying the different line types
+        # (fracture, constraint, boundary). For the Gmsh processing, a string-based
+        # system is used, as this is more readable and closer to the system employed in
+        # Gmsh. In practice, this requires translating from GmshInterfaceTags values to
+        # names.
+        # In addition to this translation, the below code also does some interpretation
+        # of the information obtained during geometry processing.
+
         decomp = self._decomposition
 
         edges = decomp["edges"]
-
-        # Only lines specified as fractures are defined as physical
-        phys_line_tags = {}
+        # Information about line types is found in the third row of edges
         edge_types = edges[2]
+
+        # Process information about lines that should be tagged as physical by Gmsh.
+        # These are fractures, domain boundaries and auxiliary (constraints).
+        # phys_line_tags is a mapping from line index to the Tag.
+        phys_line_tags: dict[int, GmshInterfaceTags] = {}
```

Comment:
Isn't it a bit strange that auxiliary is "physical"?

Reply:
The physical is `gmsh` terminology; I am afraid we'll have to do translation between terminologies somewhere.